src/main/java/org/broad/igv/prefs/IGVPreferences.java

```diff
@@ -248,9 +249,9 @@ public boolean getAntiAliasing() {
     private void updateCaches(String key, String value) {
         key = key.trim();
         if (booleanCache.containsKey(key)) {
-            booleanCache.put(key, new Boolean(value));
+            booleanCache.put(key, Boolean.valueOf(value));
         }
-        colorScaleCache.remove(key);
+        colorScaleCache.remove(key); //TODO color scale cache doesn't use key strings so something here is wrong.
```

Comment:
Intellij pointed out that these keys are the wrong type so it seems very likely that this cache isn't behaving as expected.  We should probably address that seperately.

Reply:
OK I opened an issue and assigned to myself